mk

```diff
@@ -339,6 +339,80 @@ GOMOBILE = $(shell go env GOPATH)/bin/gomobile
 OONIMKALL_V := $(shell date -u +%Y.%m.%d-%H%M%S)
 OONIMKALL_R := $(shell git describe --tags || echo '0.0.0-dev')
 
+#help:
```

Comment:
leaving here a couple alternatives to make that I've got in my radar (you probably know them already):

* https://github.com/fugue/zim/
* https://magefile.org/